packages/webview_flutter/lib/src/tizen_webview_controller.dart

```diff
@@ -0,0 +1,390 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_tizen/widgets.dart';
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart';
+import 'tizen_webview.dart' as tizen_webview;
+
+/// An implementation of [PlatformWebViewController] using Flutter for Web API.
+class TizenWebViewController extends PlatformWebViewController {
+  /// Constructs a [TizenWebViewController].
+  TizenWebViewController(super.params)
+      : _webview = tizen_webview.TizenWebview(),
+        super.implementation();
+
+  final tizen_webview.TizenWebview _webview;
+
+  /// Called when [TizenView] is created.
+  void onCreate() {
+    _webview.onCreate();
+  }
+
+  @override
+  Future<void> loadFile(String absoluteFilePath) async {
+    assert(absoluteFilePath != null);
+    _webview.loadFile(absoluteFilePath);
+  }
+
+  @override
+  Future<void> loadFlutterAsset(String key) async {
+    assert(key.isNotEmpty);
+    _webview.loadFlutterAsset(key);
+  }
+
+  @override
+  Future<void> loadHtmlString(
+    String html, {
+    String? baseUrl,
+  }) async {
+    assert(html != null);
+    _webview.loadHtmlString(html, baseUrl: baseUrl);
+  }
+
+  @override
+  Future<void> loadRequest(LoadRequestParams params) async {
+    if (!params.uri.hasScheme) {
+      throw ArgumentError(
+          'LoadRequestParams#uri is required to have a scheme.');
+    }
+
+    switch (params.method) {
+      case LoadRequestMethod.get:
+        _webview.loadRequest(params.uri.toString());
+        return;
+      case LoadRequestMethod.post:
+        break;
```

Comment:
Post requests are not supported?

Reply:
The webview_flutter_tizen(ewk) did not support post request.
I've found an api related to this, but haven't yet checked that it works well.
After this patch is merged, I will make a separate PR.